tools/pindel/pindel.xml

```diff
@@ -21,7 +21,7 @@
         -o1 $outputRawFile
         -o2 $outputVcfFile
 
-        --number_of_procs \${GALAXY_SLOTS:-4}
+        --number_of_threads \${GALAXY_SLOTS:-4}
```

Comment:
It seems like there is a diminishing return for using threads in Pindel. The procs split up the chromosome and run each independently, and that seems to scale to # of chromes. But threads only seems to speed things up to 4 threads. 
It would probably be best to allocate 2 threads and GALAXY_SLOTS / 2 procs